frontend/components/LineChart.tsx

```diff
@@ -104,28 +222,41 @@ const LineChart: React.FC<Props> = ({ dataSource }) => {
             }}
           >
             <div className="mb-1 text-yellow-500">
-              Total cost of the first <b>{slice.points[0].data.xFormatted}</b>{" "}
-              month(s).
+              Total cost of the first <b>{point.data.xFormatted}</b> month(s).
             </div>
-            {slice.points
-              .sort((a, b) => Number(b.data.y) - Number(a.data.y))
-              .map((point) => (
+            {hasMoreHigherPoints && <div>...</div>}
+            {higherPoints.map((point) => (
+              <div key={point.id} className="flex justify-between items-center">
+                <div className="flex items-center">
+                  <span>{point.id}</span>
+                </div>
+                <b className="ml-2">{withComma(point.y)} $</b>
+              </div>
+            ))}
+            <div
+              key={point.id}
+              className="flex justify-between items-center my-1"
+            >
+              <div className="flex items-center">
                 <div
-                  key={point.id}
-                  className="flex justify-between items-center"
-                >
-                  <div className="flex items-center">
-                    <div
-                      className="w-2 h-2 mr-2"
-                      style={{
-                        backgroundColor: point.color,
-                      }}
-                    ></div>
-                    <span>{point.serieId}</span>
-                  </div>
-                  <b className="ml-2">{point.data.yFormatted}</b>
+                  className="w-2 h-2 mr-2"
+                  style={{
+                    backgroundColor: point.color,
+                  }}
+                ></div>
+                <span>{point.serieId}</span>
```

Comment:
Might be a typo. `seriesId`?

Reply:
This is a typo in the chart library. 🤣
![Shot-2022-10-26-Code-fLutHAgT](https://user-images.githubusercontent.com/56376387/197922274-6231b2b1-6088-4089-baa5-e14d0c435027.png)

Reply:
lol